rocky/rocky/middleware/auth_token.py

```diff
@@ -9,12 +9,13 @@ def middleware(request):
         if not request.user.is_authenticated and "authorization" in request.headers:
             authenticator = TokenAuthentication()
             try:
-                user, token = authenticator.authenticate(request)
+                user_and_token = authenticator.authenticate(request)
```

Comment:
Why not just use _ for the token as it isn't used anywhere in this context?

Reply:
That would have been better in the old code, but we can't unpack the return value if it is None, so we have to assign it to a variable first to prevent the exception.